apps/wallet-mobile/src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -51,23 +52,25 @@ const ErrorView = ({state}: {state: State}) => {
 
   return (
     <View style={styles.root}>
+      <Spacer height={70} />
```

Comment:
Do we still have to mix Space and Spacer?

Reply:
yap we do :(, we will be able to get rid of it, only when creating our design system components.